geoh5py/objects/drillhole.py

```diff
@@ -333,10 +349,11 @@ def add_data(
 
             attr["name"] = name
 
-            tolerance = 1e-2
             if "tolerance" in list(attr.keys()):
                 assert attr["tolerance"] > 0, "Input depth 'tolerance' must be >0."
                 tolerance = attr["tolerance"]
+            else:
+                tolerance = self.tolerance
```

Comment:
**suggestion (test):** have a test on using this tolerance when adding data  (test for one data added, and one replaced)

BTW what is the expected behavior: is data replaced if within the tolerance or ignored?

Reply:
No replacement of the data value itself, the entry just gets inserted at a matching existing depth. So I guess, in a way, depth is rounded to the matching value yes.

Reply:
because is is a new drillhole log, not adding entries into an existing log. right